### DISCUSSION TITLE
Is there a way to use the same scanned files when installing nicotine-plus-git in Manjaro?

### DISCUSSION THREAD

**Discussion by ghost** [nicotine-plus/nicotine-plus#2112] [Q&A]
I have `nicotine+` installed in Manjaro, and [I've been told](https://github.com/nicotine-plus/nicotine-plus/issues/2098) to try a new version to see if it fixes my connection problem. I would like to use the files from the `nicotine+` package with `nicotine-plus-git (3.3.0)` so I don't have to re-scan all files. How can I do that? Maybe I can modify the [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=nicotine-plus-git) or symlink a folder to accomplish this?

**Comment by slook** [comment]
What is "`nicotine-plus-git`"? If that is some package provided by your distro's community then it won't be as up-to-date as the packages available directly from the Nicotine+ repo ([3.3.0dev1](https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md) or [3.2.3rc2](https://github.com/nicotine-plus/nicotine-plus/blob/3.2.x/doc/TESTING.md)) which are updated after every commit.

> How can I do that?

You just need to get or clone an unstable package and then run `./nicotine` without any options., then the program would normally just use your existing user-data and `config` file that is already stored in your home folder.

Your existing settings including shared files and stored history etc shall be used whether you run either a stable or unstable Nicotine+ package or you run Nicotine+ from a local Git folder ([3.3.0dev1](https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md#git) or [3.2.3rc2](https://github.com/nicotine-plus/nicotine-plus/blob/3.2.x/doc/TESTING.md#git)), unless you use a runtime command line option (`--user-data` and/or `--config`) that specifies an alternate data folder or config file (see `./nicotine --help` or `man nicotine`).

> Maybe I can modify the [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=nicotine-plus-git) or symlink a folder to accomplish this?

There should not be any need for you to have to deal with any packaging procedures, unless you want to help your distro's community with stable distribution [package maintenance](https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/PACKAGING.md).

If the `nicotine-plus-git` package in your community repo does not behave as expected, then please inform the AUR package maintainer about it.

> I don't have to re-scan all files

Re-scanning your shared files shouldn't be a big deal, unless you switch back and forth between versions prior to 3.2.0 in which case a full scan is done instead of a quick rescan (which might take minutes instead of seconds).

> try a new version to see if it fixes my connection problem

The commit for the fix you want to test is included in both branches ("[3.2.x](https://github.com/nicotine-plus/nicotine-plus/commit/3fa098244650263fb3f69c71746cb9ce549a8138)" and "[master](https://github.com/nicotine-plus/nicotine-plus/commit/2728cb86c90b97f8bf39c6ec7c74320203047a45)"). You can switch between any versions without loosing your `config` settings (although it's always a good idea to keep a backup of your "`~/.nicotine/*.*`" user-data folder).

Debugging and development usually happens in the `master` branch (3.3.0), but at the moment we are focused on testing the stability and performance of the `3.2.x` branch because the stable release of Nicotine+ 3.2.3 is imminent, so testing a [current unstable build of Nicotine+ 3.2.3rc2](https://github.com/nicotine-plus/nicotine-plus/blob/3.2.x/doc/TESTING.md) would be most appreciated :)

To run Nicotine+ 3.2.x directly from a local [Git](https://git-scm.com/) folder, ensure the [runtime dependencies](../blob/master/doc/DEPENDENCIES.md) are installed, and run the following:

```sh
git clone -b 3.2.x https://github.com/nicotine-plus/nicotine-plus.git
cd nicotine-plus
./nicotine
```
